spec/features/adventures/user_edits_adventure_spec.rb

```diff
@@ -80,8 +92,18 @@
 
   scenario "authenticated user successfully adds a address to an adventure" do
     user = FactoryGirl.create(:user)
-    bucket_list_sign_in(user)
+    sign_in(user)
+    bucket_list = FactoryGirl.create(
+      :bucket_list,
+      user_id: user.id,
+      title: "Second Bucket List"
+    )
     adventure = FactoryGirl.create(:adventure, address: nil, user_id: user.id)
+    bucket_list_adventure = FactoryGirl.create(
```

Comment:
Useless assignment to variable - `bucket_list_adventure`. Did you mean `bucket_list`?